### PR TITLE
Feature/dockerize

### PR DESCRIPTION
**Pull request by ewpreston**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ lti-tools:
 # on one machine. The IP address will need to be changed to the
 # one currently assigned to the development machine.
   environment:
-    HOST_ENTRY: 10.37.145.131  mylearn.int.bbpd.io
+    HOST_ENTRY: 10.103.124.51  mylearn.int.bbpd.io
```
Comment (ewpreston) [inline docker-compose.yml]: I'm puzzled why we should have this in our main config, and not document it that you should add a HOST_ENTRY if you need it

File: Dockerfile [hunk 2]
```diff
@@ -10,7 +10,6 @@ RUN npm install
 
 # Bundle app source
 COPY . /usr/app
-RUN npm run postinstall
```
Comment (brianburdan) [inline Dockerfile]: What was this doing that we don't need anymore?

Comment (ewpreston) [reply Dockerfile]: I have no idea; this was last set in 2017; it doesn't exist; I was hoping to get Al on this review

Comment (adgilmore) [reply Dockerfile]: I don't remember specifically but it was completing the setup of something. I know, really vague but I have slept since then.

Comment (adgilmore) [reply Dockerfile]: I probably had something setup wrong so it needed to be cleaned up.

**Pull request merged**